README.md

```diff
@@ -29,6 +10,24 @@ This module is installed via npm:
 $ npm install autoscale-desired-count
 ```
 
+## Usage
+
+```js
+const getCount = require('autoscale-desired-count');
+
+const opts = {
+  concurrency: 1, // the number of concurrent jobs the worker processes
+  deadline: 60, // the time practically all jobs should be completed within
+  meanProcessTime: 10, // mean job processing time
+  requiredCompletionPercentile: 99, // percentile of jobs that should happen within the deadline
+  stddev: 5, // standard deviation (1σ) of job processing time. ie. 68% completed within ± this
+};
+
+const jobs = 500;
+const count = getCount(jobs, opts);
+// count = the number of workers needed to process jobs based on passed in opts
+```
+
```

Comment:
Again, kind of nice to have a proper API section that lists types of things as per if you jsdoc the functions and then run `jsdoc2markdown`?

Reply:
Less critical here I guess becuase the `count` is clearly an int?  Probably?  Well there you go, is it? :)